Bug: When the rule fails, subsequent diff error occurs

```
app.js:1 Error: Null input. (diff_main)
    at diff_match_patch.diff_main (plugin:obsidian-linter:88:15)
    at LinterPlugin.runLinterEditor (plugin:obsidian-linter:23652:25)
    at Object.saveCommandDefinition.callback (plugin:obsidian-linter:23555:18)
    at BB (app.js:1:1464176)
    at e.executeCommand (app.js:1:1465540)
    at e.onTrigger (app.js:1:1342060)
    at e.handleKey (app.js:1:1019276)
    at t.e.handleKey (app.js:1:1019389)
    at t.handleKey (app.js:1:1019598)
    at e.onKeyEvent (app.js:1:1020514)
```

--- a/src/main.ts
+++ b/src/main.ts
@@ -341,6 +341,7 @@ export default class LinterPlugin extends Plugin {
       newText = this.rulesRunner.lintText(createRunLinterRulesOptions(oldText, file, this.momentLocale, this.settings));
     } catch (error) {
       this.handleLintError(file, error, 'Lint File Error in File \'${file.path}\'', false);
+      return;
     }
 
     // Replace changed lines

